docs-app/.prettierrc.js

```diff
@@ -1,9 +1,6 @@
 'use strict';
 
 module.exports = {
-  printWidth: 80,
-  singleQuote: true,
-
```

Comment:
I didn't realize before that setting `singleQuote` to `true` here affects all file extensions. It may be better to override `prettier`'s default rules on a case-by-case basis.

https://github.com/ember-cli/ember-cli/blob/v4.12.1/blueprints/app/files/.prettierrc.js